python-api/gstswitch/helpers.py

```diff
@@ -87,7 +90,7 @@ def audio_port(self, audio_port):
     @property
     def running_tests_video(self):
         """Get the currently running test video list"""
-        return self._running_tests_video
+        return list(self._running_tests_video)
```

Comment:
Should you be returning a tuple here? This property shouldn't be modifiable right?